lighthouse-cli/bin.ts

```diff
@@ -229,11 +235,32 @@ function lighthouseRun(addresses: Array<string>, config: Object, flags: Object)
   return lighthouse(address, flags, config)
     .then((results: Results) => Printer.write(results, outputMode, outputPath))
     .then((results: Results) => {
+      const filename = `${assetSaver.getFilenamePrefix({url: address})}.report.html`;
+
       if (outputMode === Printer.OutputMode[Printer.OutputMode.pretty]) {
-        const filename = `./${assetSaver.getFilenamePrefix({url: address})}.report.html`;
         Printer.write(results, 'html', filename);
       }
 
+      // Generate report.html, host it and open it in the default browser
+      if (flags.view) {
+        performanceXServer.startServer(0).then((port: number) => {
+          const filePath = `${performanceXServer.FOLDERS.REPORTS}/${filename}`;
+          const htmlExist = outputMode === Printer.OutputMode[Printer.OutputMode.pretty]
+            || (outputPath !== 'stdout' && outputMode === Printer.OutputMode[Printer.OutputMode.html]);
```

Comment:
Sure. I'm happy to regenerate the report rather than read it from where we just printed it on disk.

if --view becomes a default, then the automatic disk-save won't really make sense.